test/unit/unit-entry.js

```diff
@@ -200,7 +200,7 @@ const TEST_GLOBAL = {
   },
   EventEmitter,
   ShellService: {isDefaultBrowser: () => true},
-  FilterExpressions: {eval() { return Promise.resolve(true); }},
+  FilterExpressions: {eval() { return Promise.resolve(false); }},
```

Comment:
The fix from yesterday landed before I realised the mistake. The test was incorrect, we should default to failing in our mocks.

Reply:
good idea